### PR TITLE
btcpay version option

### PR DESCRIPTION
**Pull request by Kukks**
Allow a user to specify their own docker image for btcpayserver via `BTCPAY_VERSION`

### CODE REVIEW

**Comment by NicolasDorier** [comment]
can you rename to `BTCPAY_IMAGE`?

**Comment by Kukks** [comment]
yeah one sec

**Comment by Kukks** [comment]
I haven't tested this one yet @NicolasDorier

**Comment by Kukks** [comment]
tested it now :) gogogo

File: docker-compose-generator/docker-fragments/btcpayserver.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
 
   btcpayserver:
     restart: unless-stopped
-    image: nicolasdorier/btcpayserver:1.0.3.9
+    image: ${BTCPAY_IMAGE:-nicolasdorier/btcpayserver:nicolasdorier/btcpayserver:1.0.3.9}
```
Comment (NicolasDorier) [inline docker-compose-generator/docker-fragments/btcpayserver.yml]: it broke `${BTCPAY_IMAGE:-nicolasdorier/btcpayserver:nicolasdorier/btcpayserver:1.0.3.9}` double tag

**Pull request merged**